test/e2e/utils/common.js

```diff
@@ -105,11 +105,11 @@ const testValues = {
         day: '2',
         year: '2012'
       },
-      childDisabledBefore18: false,
-      childAttendedSchoolLastYear: false,
+      childDisabledBefore18: '',
+      childAttendedSchoolLastYear: '',
       childEducationExpenses: '6000',
-      childCohabitedLastYear: false,
-      childReceivedSupportLastYear: false,
+      childCohabitedLastYear: '',
+      childReceivedSupportLastYear: '',
```

Comment:
I think this file needs to be updated on lines 361 and onward, for the `completeChildInformation` section on what inputs to click.